### ISSUE TITLE
Allow excluding directories in "Find in files"

### ISSUE THREAD

**Issue by brigantino2** [Komodo/KomodoEdit#794]
As suggested in this forum thread
http://forum.komodoide.com/t/exclude-folder-from-find-in-files/2199
I'd like to make a request for improvement.
If you open the Find dialog and you make a "files" search, there is currently no way of adding a folder to "Exclude" list. In particular, I need to exclude certain folders that include static files, that would otherwise pollute my searches (a lot).
Thanks

**Comment by Naatan** [comment]
Renamed it to be less specific, ie. we want to support this anywhere we support excluding files.

Thanks for filing this :)

**Comment by cgchoffman** [comment]
This works as expected for me.  I know Publishing does this so I just did a simple test and it works fine.  

folders:
testing-> _>yup > morepizza.py
              \_>nope > pizza.py

I did a search in `testing` for a word contained in both files.  I added `nope` to the excludes.  I only get the string in the `yup` folder.

Does appear there is anything to do here.

**Comment by cgchoffman** [comment]
@brigantino2 what OS are you on?  Can you do a simple test like mine and see if you still can't do excluding of folders?  If you CAN, we'll need to figure out why it wasn't working for you the first time.

**Comment by brigantino2** [comment]
Hi Carey,

I'm on linux Mint. I've tried, but if you place just the name of the 
folder in the "exclude" list, it will exclude all folders with that 
name, while I need to exclude a specific one (other folders are present 
with the same name elsewhere: 'static', or 'media' for example).
If I use the full path of the folder, it doesn't filter anything.

Il 04/11/2015 18:28, Carey Hoffman ha scritto:
>
> @brigantino2 <https://github.com/brigantino2> what OS are you on? Can 
> you do a simple test like mine and see if you still can't do excluding 
> of folders? If you CAN, we'll need to figure out why it wasn't working 
> for you the first time.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Komodo/KomodoEdit/issues/794#issuecomment-153800556>.
>

-- 

Alessio Moretti

/Address/ · via del Ponte alle Mosse 73, 50144 Firenze

/Tax Code/ · MRTLSS75S13D612A

/VAT n. · 06274950481/

//

**Comment by cgchoffman** [comment]
Yes, i read the ticket wrong.  Please ignore those emails with my comments.  We've removed them from this ticket.  Make sure you open the ticket in your browser when you reply.

**Comment by stramin** [comment]
I need this feature too I can add a folder, but I can not add a subfolder, something like "inc/files/" doesn't work.

**Comment by stramin** [comment]
Hi again, still having this issue on Komodo 11, I want to exclude a folder called dompdf, but when I write "dompdf" it excludes a file called dompdf.php outside the folder :/
